packages/database/migration/20250701163837_add_ids_to_matrix_questions/migration.sql

```diff
@@ -0,0 +1,56 @@
+-- Migration to add unique IDs to Matrix question rows and columns
```

Comment:
There are a few issues here:

1. We’re manually creating the cuid2, which may look different from the automatically generated one after this PR.
2. We’re not essentially updating the schema with this, but rather the existing data. Therefore, it should be done via a data migration.